### PR TITLE
Update password Option should be added in the login fragement.

### PR DESCRIPTION
**Pull request by Aditya185**
AC-574 - https://issues.openmrs.org/AC-574

## Description of what I changed

I have added Change Password Option in login Fragment.
i have added ChangePassword.java file in activities.

i have added only added visual part till now ..if you approve this i will then work on connecting this with rest api.

## Issue I worked on

AC-574

yes

JIRA Issue: https://issues.openmrs.org/browse/AC-574
Screenshot_20190331-160548
Screenshot_20190331-160613

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [ ] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by Aditya185** [comment]
when i was creating the issue the android client project was not found

**Comment by deepak140596** [comment]
@Aditya185 , head over [this link](https://issues.openmrs.org/projects/AC/issues/AC-480?filter=allopenissues) to see/open all the issues related to this project

**Comment by Aditya185** [comment]
how to create an issue please explain in android client

**Comment by deepak140596** [comment]
Head over to [this link](https://wiki.openmrs.org/display/docs/Getting+Started+as+a+Developer) to get started.

**Comment by Aditya185** [comment]
When I click on create issue only two projects
 are shown.....

**Comment by Aditya185** [comment]
@ I have created the case,but OpenMRS has not yet reply for giving me access to JIRA

**Comment by vansha10** [comment]
@Aditya185 Please log out and log in again.

**Comment by Abdelaty** [comment]
I didn't found the Issue on Jira Issues, so you have to follow the[ PR guidelines.](https://wiki.openmrs.org/display/docs/Pull+Request+Tips)

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ChangePassword.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
+package org.openmrs.mobile.activities;
+
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.support.design.widget.TextInputLayout;
```
Comment (sanjulamadurapperuma) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ChangePassword.java]: "android.support.design.widget does not exist
import android.support.design.widget.TextInputLayout"

This is the error shown by the Compilers: Please check if there are any compilation errors before committing.

**Comment by f4ww4z** [comment]
@Aditya185 See my comment on JIRA. Address it first.

**Pull request closed without merging**